src/lib/holocene/input/input.svelte

```diff
@@ -187,24 +185,24 @@
 <style lang="postcss">
   /* Base styles */
   .input-container {
-    @apply surface-input relative box-border inline-flex h-10 w-full items-center rounded border border-subtle text-sm;
+    @apply surface-primary relative box-border inline-flex h-10 w-full items-center rounded-lg border-2 border-subtle text-sm focus-within:outline-none focus-within:ring-4 focus-within:ring-primary/70;
 
     &.error,
     &.invalid {
-      @apply border-error focus-within:shadow-danger/50;
+      @apply border-danger focus-within:ring-danger/70;
 
       > .input {
-        @apply caret-danger;
+        @apply caret-red-700;
```

Comment:
maybe we add a `caret-danger` in the plugin?

Reply:
[Done](https://github.com/temporalio/ui/pull/2118/commits/e9c28d705be0987e503d4781b35e646942fa7679) ✅